### PR TITLE
Update the documentation name, icon and tooltip

### PR DESCRIPTION
**Pull request by devramsean0**
## Summary of the problem

See #5342 

## Describe your changes

I updated the icon, name and tooltip in the nav bar. I also changed the title of the Documentation page to match

image

### CODE REVIEW

File: app/views/events/_nav.html.erb [hunk 1]
```diff
@@ -90,6 +90,7 @@
         data: { tour_step: "perks" },
         disabled: @event.hardware_grant?,
         selected: local_assigns[:selected] == :promotions unless !@event.approved? %>
+      <%= dock_item "Documents",
```
Comment (garyhtou) [inline app/views/events/_nav.html.erb]: This looks like invalid ERB

File: app/views/events/_nav.html.erb [hunk 2]
```diff
@@ -98,8 +99,8 @@
         selected: local_assigns[:selected] == :google_workspace %>
       <%= dock_item "Documentation",
         event_documentation_path(event_id: @event.slug),
-        tooltip: "Commonly asked questions and how-tos",
-        icon: "info",
+        tooltip: "See legal documents, helpful tips, account statements, and FAQ",
```
Comment (dwahbe) [inline app/views/events/_nav.html.erb]: I know I suggested this but I would update this to 
```suggestion
        tooltip: "View legal documents, account statements, and FAQ",
```

Comment (sarthaktexas) [reply app/views/events/_nav.html.erb]: I concur with Dylan

File: app/views/events/_nav.html.erb (hunk 1 above)
Comment (devramsean0) [reply app/views/events/_nav.html.erb]: It appears to be valid to me,
Where do you think the error is?

File: app/views/events/_nav.html.erb (hunk 2 above)
Comment (devramsean0) [reply app/views/events/_nav.html.erb]: I also concur

File: app/views/events/_nav.html.erb (hunk 1 above)
Comment (sarthaktexas) [reply app/views/events/_nav.html.erb]: Screenshot 2024-03-05 at 3 30 53 PM
be so fr

**Review by sarthaktexas — CHANGES_REQUESTED** [review]

**Comment by sarthaktexas** [comment]
I sent you a corrected nav bar file Sean!

**Review by sarthaktexas — APPROVED** [review]

**Pull request merged**